pkg/cli/admin/inspect/pod.go

```diff
@@ -142,6 +155,19 @@ func (o *InspectOptions) gatherContainerRotatedLogFiles(destDir string, pod *cor
 				return
 			}
 
+			// make sure rotated logs honor --since and --since-time
+			// when set by the user.
+			if parts := strings.Split(fileName, "."); len(parts) >= 3 {
```

Comment:
I wonder about can we extract the dates via plain regex instead of `Split`s?

Reply:
that was my initial approach but I could not easily produce a regex that matched only the date part in the filename (there is more than one numeric part to it and I'm not very good with regex). 
if you're willing to help me out we can definitely change it.

Reply:
What about something like this?;
```go
re := regexp.MustCompile(`\.(\d+-\d+)\.gz$`)
match := re.FindStringSubmatch(filename)

if len(match) != 2 {
	return "", fmt.Errorf("No match found")
}

return match[1], nil
```

Reply:
Thanks for the regex! nice to know how it can be done.
I agree with you about keeping the `Split`, it kinda looks dumb but that can be a strength (easy to understand).